### ISSUE TITLE
Move this repository to "theupdateframework"

### ISSUE THREAD

**Issue by thaJeztah** [docker/notary-official-images#13]
With notary itself being in that organization, it probably makes sense to move this repository as well

@justincormack @endophage

**Comment by endophage** [comment]
I'd suggest this is left here for posterity and consistency of already published versions. New versions can be published to a new repo in theupdateframework.

**Comment by thaJeztah** [comment]
@endophage if we move the repo, GitHub would redirect to the new location, in which case the history would not get lost, so (my €0.02) moving the repo would be the better approach 🤗

**Comment by justincormack** [comment]
Yes I think that moving is more consistent as the history will then be in one place.

**Comment by endophage** [comment]
My opinion here is weakly held so feel free to go ahead. My thinking was there isn't really a good way as a downloader to verify _why_ there's a redirect. Leaving this repo in place and updating the README to say "go here for new releases" provides a more trustworthy transfer of owner.
